### PR TITLE
chore: index.css -> separate .scss files

### PR DESCRIPTION
**Pull request by broofa**
No actual changes to styling.  Just refactoring code to take advantage of Parcel's (S)CSS support

* Switch to SCSS, using notation where appropriate
* Separate index.css into component-specific *.scss files
* `stylelint` for CSS linting (included in `npm test`)
* (... and moved favicon.* images to `images` directory to clean up root directory a little)

### CODE REVIEW

File: .stylelintrc.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "stylelint-config-standard"
+}
```
Comment (fregante) [inline .stylelintrc.json:3]: I'd add EOL where missing

**Review by fregante — APPROVED** [review]

File: .stylelintrc.json (hunk 1 above)
Comment (broofa) [reply .stylelintrc.json:3]: End-of-file newlines [are enabled](https://github.com/stylelint/stylelint-config-standard/blob/master/index.js#L78).  ... or were you referring to some other missing EOLs?

**Comment by broofa** [comment]
FYI, checks are breaking because parcel part of the CSS sourcemap changes on every build (even when source doesn't change. 😢 )  Looking into it.

File: .stylelintrc.json (hunk 1 above)
Comment (fregante) [reply .stylelintrc.json:3]: eslint doesn’t lint JSON files. This file lacks a EOL newline

**Pull request merged**